lib/chewy/config.rb

```diff
@@ -40,7 +40,19 @@ class Config
 
       # Where Chewy expects to find index definitions
       # within a Rails app folder.
-      :indices_path
+      :indices_path,
+
+      # Set index refresh_interval setting to -1 before reset and put the original value after.
+      # If setting not present, put back to default 1s
+      # https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html
+      :reset_disable_refresh_interval,
+
+      # Set number_of_replicas to 0 before reset and put the original value after
+      # https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html
+      :reset_no_replicas,
+
+      # Refresh or not when import async (sidekiq, resque, activejob)
+      :disable_refresh_async
```

Comment:
Hey, why did you add this?
I'm thinking about some options passed to the block: `Chewy.strategy(:sidekiq, refresh: false) { do_stuff }` Instead of the global option. However it would not affect anything, so I'm ok to accept it if you don't want to change your mind.